app/views/layouts/application.html.erb

```diff
@@ -16,7 +16,5 @@
     <%= render 'shared/navbar' %>
     <%= render 'shared/flashes' %>
     <%= yield %>
-    <p class="notice"><%= notice %></p>
```

Comment:
Notice and alert class have been removed, could you give some details ?

Reply:
The notice and alert are included in the shared / flashes.

Reply:
Indeed ! very nice job! 👯